<?xml version="1.0"?>
<launch>
    <node pkg="mursim_actuator" type="actuator_node" name="mursim_actuator" output="screen"/>
    <param name="vehicle_model" value="gotthard"/>
    <param name="equal_drive" value="true"/>
</launch>
